core/src/main/java/com/google/instrumentation/stats/MutableView.java

```diff
@@ -19,8 +19,10 @@
 import com.google.instrumentation.stats.ViewDescriptor.DistributionViewDescriptor;
 import com.google.instrumentation.stats.ViewDescriptor.IntervalViewDescriptor;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
nit: LinkedHashMap if nothing else, but for tests getting consistent ordering regardless of JRE. (not necessarily your tests, anyone who uses this in tests)